recipes/semchunk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "semchunk" %}
+{% set version = "2.2.0" %}
+{% set build_number = 0 %}
+{% set python_min = "3.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/semchunk-{{ version }}.tar.gz
+  sha256: 4de761ce614036fa3bea61adbe47e3ade7c96ac9b062f223b3ac353dbfd26743
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: {{ build_number }}
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    # Cannot use optional dependencies like the ones mpire[dill] requires, so setting 3.11 as the floor
+    - python >={{ python_min }}
+    - tqdm
+    - mpire
+    # Required by mpire[dill]
+    - multiprocess >=0.70.15
```

Comment:
In the upstream `setup.py` (https://github.com/sybrenjansen/mpire/blob/master/setup.py#L27-L28) it reads:

```
'dill': ['multiprocess; python_version<"3.11"',
           'multiprocess>=0.70.15; python_version>="3.11"'],
```

Doesn't this suggest that you can leave the version pinned for `multiprocess` since it satisfies both py>=3.11 and py<3.11 and then relax the `python_min` here (i.e remove the local definition and just use the global conda-forge pin)?